composer.json

```diff
@@ -11,12 +11,27 @@
 		}
 	],
 	"require": {
-		"silverstripe/framework": "3.*",
-		"silverstripe/cms": "3.*"
+		"silverstripe/framework": "^4@dev",
+		"silverstripe/cms": "^4@dev"
 	},
+	"require-dev": {
+		"phpunit/PHPUnit": "^5.7"
+	},
+	"autoload": {
+        "psr-4": {
+            "SilverStripe\\IFrame\\": "code/"
+        }
+    },
+    "autoload-dev": {
```

Comment:
Sorry to contradict @robbieaverill , but this does need to be in non-dev autoload.